epsagon/wrappers/django.py

```diff
@@ -0,0 +1,99 @@
+"""
+Wrapper for Django.
+"""
+from __future__ import absolute_import
+
+import time
+import traceback
+import warnings
+import epsagon
+import epsagon.trace
+import epsagon.triggers.http
+import epsagon.runners.django
+
+from epsagon.common import EpsagonWarning
+from .http_filters import ignore_request
+
+
+class DjangoMiddleware(object):
+    """
+    Represents a Django Middleware for Epsagon instrumentation.
+    """
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+        self.runner = None
+        self.request = None
+        self.response = None
+
+        self.ignored_request = False
+
+    def __call__(self, request):
```

Comment:
There is no example on how to use the Django wrapping, I'm not sure why using `__init__` is not good enough as in flask (consider adding an example to the readme..)

Reply:
Forgot about it, thanks.